package.json

```diff
@@ -30,7 +30,7 @@
 		"@types/mocha": "^5.2.7",
 		"@types/node": "^12.11.7",
 		"json-schema-ref-parser": "^7.1.3",
-		"mocha": "^7.1.1",
```

Comment:
We also need to update the `version` field in package.json to release a new version of this package to npm